_posts/0000-01-02-shanearnott.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Part Deux"
+---
+## Check, check it out, what's it all about?
```

Comment:
# Response
## Issues
- One
- Two
- Seven

## TODO
- [ ] Not done
- [x] done

Reply:
Yeah, nah 
# _*RESOLVED*_